src/EventCard.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import img from "./parasite.png";
+
+const Event = () => {
+  return (
+    <div className="max-w-sm rounded overflow-hidden shadow-lg">
+      <img className="w-full" src={img} />
+      <div>
+        <p>
+          <span class="inline-block text-gray-700 font-semibold">
+            {" "}
+            Megascreening! Parasite by Bon...
+          </span>
+        </p>
+        <p>
+          <span class="inline-block text-gray-700"> Sat, Feb 22 </span>
+        </p>
+        <p>
+          <span class="inline-block text-gray-700"> 1599 interested </span>
```

Comment:
Do you need these `<p>` tags? Try just using the `<span>`s by themselves without wrapping them and then get back to me.